src/GraphQL/Field/Root/Entity/EntityQueryField.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Drupal\graphql\GraphQL\Field\Root\Entity;
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Entity\TypedData\EntityDataDefinitionInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\graphql\GraphQL\Field\FieldBase;
+use Drupal\graphql\Utility\StringHelper;
+use Fubhy\GraphQL\Type\Definition\Types\ModifierInterface;
+use Fubhy\GraphQL\Type\Definition\Types\Type;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+use Youshido\GraphQL\Execution\ResolveInfo;
+use Youshido\GraphQL\Type\Enum\EnumType;
+use Youshido\GraphQL\Type\ListType\ListType;
+use Youshido\GraphQL\Type\Scalar\AbstractScalarType;
+use Youshido\GraphQL\Type\Scalar\IntType;
+use Youshido\GraphQL\Type\TypeInterface;
+
+class EntityQueryField extends FieldBase implements ContainerAwareInterface {
+  use ContainerAwareTrait;
+
+  /**
+   * The entity type handled by this field instance.
+   *
+   * @var string
+   */
+  protected $entityTypeId;
+
+  /**
+   * Constructs an EntityQueryField object.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeInterface $entityType
+   *   The entity type handled by this field instance.
+   * @param \Youshido\GraphQL\Type\TypeInterface $outputType
+   *   The output type of the field.
+   */
+  public function __construct(EntityTypeInterface $entityType, TypeInterface $outputType) {
+    $this->entityTypeId = $entityType->id();
+
+    /** @var \Drupal\Core\Entity\TypedData\EntityDataDefinition $definition */
+    $definition = \Drupal::typedDataManager()->createDataDefinition("entity:{$this->entityTypeId}");
+    $arguments = $this->getQueryArguments($definition);
+    $argumentNames = StringHelper::formatPropertyNameList(array_keys($arguments));
+
+    // Generate a human readable name from the entity type.
+    $typeName = StringHelper::formatPropertyName($this->entityTypeId);
+
+    $queryValue = new EntityQueryValue($this->entityTypeId, array_flip($argumentNames));
+
+    $config = [
+      'name' => "{$typeName}Query",
+      'type' => new ListType($outputType),
```

Comment:
We want to return a ConnectionType here instead (that's the new feature we need from Relay)